internal/test/util/pmm.go

```diff
@@ -179,10 +176,6 @@ func (p *PMM) Deploy() {
 		p.t.Fatal(err, "failed to ping PMM")
 		return
 	}
-	if err := authUntilSuccess(p.PMMURL(), p.timeout); err != nil {
-		p.t.Fatal(err, "failed to auth PMM")
-		return
-	}
```

Comment:
It has no sense since few lines before we hit `/version` endpoint that requires authentication.